tirmmer_decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator
```

Comment:
- [ ] The TrimmerDecorator should inherit from the BaseDecoratore class, kindly consider implementing this so your decorator works correctly